dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
```

Comment:
Ensure that `SessionLocal` is correctly configured to return an `AsyncSession`. This is crucial for maintaining asynchronous database operations throughout your application.